docs/partials/recording-gameplay/_data-zip-partial.js

```diff
@@ -11,6 +11,32 @@ Basic information about when this tick was captured within the context of the ov
 This includes the sequential tick number (which matches the name of the file), 
 the time in seconds since the start of the game, and the in-game timescale.
 
+### Key Frame Flags
+
+Key Frame flags indicate that important changes occurred to the game state since the previous tick was captured.
+Flags are represented as an array of strings in the \`keyFrame\` field.
+
+| Key Frame Flag              | Description                                                                                                                                                                   |
+|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| FIRST_FRAME                 | Indicates that this tick is the first tick of the recording                                                                                                                   |
+| SCENE                       | Indicates that the visible scenes are different from those in the previous tick                                                                                               |
```

Comment:
SCENE is no longer used and can be removed

GAME_ELEMENT_RENDERER_COUNT is no longer used and can be removed

UI_ELEMENT was combined with GAME_ELEMENT , is no longer used and can be combined/removed